client/src/routes/Splash/index.jsx

```diff
@@ -3,17 +3,22 @@ import { Link } from "react-router-dom";
 import styles from "./Splash.module.css";
 import Button from "../../components/Button";
 import Container from "../../components/Container";
+import Logo from "../../components/Logo";
+import Header from "../../components/Header";
 
-const { container, title, imgWrapper, img, link } = styles;
+const { logoPosition, container, title, imgWrapper, img, link } = styles;
 
 function Splash() {
   return (
     <Container>
-      {/* <Navigation /> */}
+      <Header title="" link="/home" />
```

Comment:
Is this component used mainly for vertical spacing? If it is, then I think it would be a semantic issue